lib/mastodon/cli/media.rb

```diff
@@ -151,7 +151,56 @@ def remove_orphans
           end
         end
       when :fog
-        fail_with_message 'The fog storage driver is not supported for this operation at this time'
+        fog_directory = Paperclip::Attachment.default_options[:fog_directory]
+        connection = Fog::Storage.new(Paperclip::Attachment.default_options[:fog_credentials])
+        directory = connection.directories.get(fog_directory)
+        last_key = options[:start_after]
+
+        loop do
+          objects = begin
+            directory.files.all(prefix: prefix, marker: last_key, limit: 1000)
```

Comment:
As we found out yesterday, this should be changed to the following:
```suggestion
            directory.files.all(prefix: prefix, marker: last_key, limit: 1000).to_a
```

Reply:
It's done. Haven't fully tested but I believe this is correct. Will test later and report if I notice any problems.